packages/usa-validation/src/index.js

```diff
@@ -1,14 +1,57 @@
 const behavior = require("../../uswds-core/src/js/utils/behavior");
 const validate = require("../../uswds-core/src/js/utils/validate-input");
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+
+const VALIDATE_INPUT = "input[data-validation-element]";
+const CHECKLIST_ITEM = `.${PREFIX}-checklist__item`;
 
 function change() {
   validate(this);
 }
 
-const validator = behavior({
-  "keyup change": {
-    "input[data-validation-element]": change,
+const enhanceValidation = (target) => {
+  selectOrMatches(VALIDATE_INPUT, target).forEach((input) => {
```

Comment:
The enhanceValidation() function is something we should be able to call on a single component.

Currently, if I created a dynamic validation component and ran `enhanceValidation()` I would enhance **all** validation components again. That's something we only want to run on init().

I would move the `selectOrMatches()` to `init()`.

Reply:
Moved enhanceValidation to init in https://github.com/uswds/uswds/pull/4914/commits/a57aff1446383a37b762637069c4b63f9691c52c